app/javascript/components/concept-map/PathLineSVG.tsx

```diff
@@ -1,10 +1,21 @@
 import React from 'react'
 
 import { ConceptPath } from './concept-map-types'
-import { computeBezier } from './helpers/svg-draw-helpers'
 
 export const PathLineSVG = ({ path }: { path: ConceptPath }): JSX.Element => {
-  return (
-    <path d={computeBezier(path)} className={`line-width-2 ${path.status}`} />
-  )
+  const computedBezier = React.useMemo(() => {
+    const { start, end } = path
+    const halfDeltaY = (end.y - start.y) / 2
+    const dx1 = start.x
+    const dy1 = start.y + halfDeltaY
+    const dx2 = end.x
+    const dy2 = end.y - halfDeltaY
+
+    return `M ${start.x}, ${start.y} C ${dx1}, ${dy1} ${dx2}, ${dy2} ${end.x}, ${end.y}`
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [path.start.x, path.start.y, path.end.x, path.end.y])
+
+  return <path d={computedBezier} className={`line-width-2 ${path.status}`} />
```

Comment:
I'd rather the `line-width-2` can be added via the stylesheet, rather than inlined.

I'm also not sure the `path.status` needs adding here. We can instead use the status reference that appears on the SVG itself for styling changes.

Reply:
```suggestion
  return <path d={computedBezier} />
```